src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2578,7 +2618,7 @@ class DropIndexStatementSegment(BaseSegment):
         Ref.keyword("CONCURRENTLY", optional=True),
         Ref("IfExistsGrammar", optional=True),
         Ref("IndexReferenceSegment"),
-        OneOf("RESTRICT", Ref.keyword("CASCADE", optional=True), optional=True),
+        Ref("DropBehaviorGrammar", optional=True),
```

Comment:
can't comment on it since you haven't touched it as part of this PR but if you look at the `type` of this class we need to update it to `type = "drop_index_statement"`

Reply:
✔